bindings/wasm/src/did/wasm_core_did.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity_iota::did::CoreDID;
+use wasm_bindgen::prelude::*;
+
+use crate::did::WasmIotaDID;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+/// A Decentralized Identifier (DID).
+#[wasm_bindgen(js_name = CoreDID, inspectable)]
+pub struct WasmCoreDID(pub(crate) CoreDID);
+
+#[wasm_bindgen(js_class = CoreDID)]
+impl WasmCoreDID {
```

Comment:
This should expose the `DID` trait methods, not necessary for this PR though.

Reply:
#973